skeleton/template.yaml

```diff
@@ -238,6 +244,9 @@ spec:
           vllmModelServiceContainer: ${VLLM_CONTAINER}
           vllmModelName: ${VLLM_MODEL_NAME}
           maxModelLength: ${VLLM_MAX_MODEL_LEN}
+          # for RHOAI
+          rhoaiSelected: ${{ parameters.rhoaiSelected }}
+          dsp: ${{ parameters.rhoaiDSP }}
```

Comment:
also cant find `${{ parameters.rhoaiDSP }}`, am I missing something?

Reply:
+1, I think we're missing the dependency on `rhoaiSelected` to show the `rhoaiDSP` field in the form. You can kind of see an example for what I am doing with local deployment here: https://github.com/Jdubrick/ai-lab-template/blob/dev/skeleton/template.yaml#L13-#L86 for setting up the dependency in the form

Reply:
It got removed from the template as I changed how we were deploying to namespaces, `rhoaiDSP` is no longer needed. I forgot to remove its reference here.

Reply:
Updated to remove the reference. I've also updated the PR description as I didn't realize it still mentioned the DSP

Reply:
No, I thought I got all references, but must've missed it 🤦

Reply:
Updated here. I'll need to update ai-lab-app as well after this merges